src/utils/mtev_lfu.c

```diff
@@ -350,12 +349,32 @@ mtev_lfu_get(mtev_lfu_t *c, const char *key, size_t key_len)
     struct lfu_entry *r = container_of(entry, struct lfu_entry, key);
     touch_lfu_cache_no_lock(c, r);
     pthread_mutex_unlock(&c->mutex);
```

Comment:
Can another thread perform an eviction on this line of code making the subsequent inc_64 "too late"?